integrations-and-supported-tools/fastai/scripts/Neptune_fastai_more_options.py

```diff
@@ -0,0 +1,44 @@
+import fastai
+from neptune_fastai.impl import NeptuneCallback
+from fastai.vision.all import *
+import neptune.new as neptune
+from neptune.new.types import File
+
+run = neptune.init(project='common/fastai-integration', tags= 'more options', api_token='ANONYMOUS')
+
+path = untar_data(URLs.MNIST_TINY)
+dls = ImageDataLoaders.from_csv(path)
+
+# Single & Multi phase logging
+
+# 1. Log a single training phase
+learn = cnn_learner(dls, resnet18)
+learn.fit_one_cycle(1, cbs=[NeptuneCallback(run, 'experiment')])
+learn.fit_one_cycle(2)
+
+# 2. Log all training phases of the learner
+learn = cnn_learner(dls, resnet18, cbs=[NeptuneCallback(run, 'experiment')])
+learn.fit_one_cycle(1)
+
+
+# Log model weights
+
+# 1. By default NeptuneCallback() saves and logs the best model for you automatically. 
+# You can disable it by setting `save_best_model` arg to False.
+
+# 2. Log Every N epochs
+n = 1
+learn = cnn_learner(dls, resnet18, cbs=[NeptuneCallback(run, 'experiment', save_model_freq=n)])
+learn.fit_one_cycle(1)
+
+# 3. Add SaveModel Callback
+learn = cnn_learner(dls, resnet18, cbs=[SaveModelCallback(), NeptuneCallback(run, 'experiment')])
```

Comment:
I am not sure how is this neptune-releated

Reply:
Actually, this is how users normally checkpoint models and other integrations work this way(azureCallback and WandbCallback). 

Furthermore, in other integrations we actually depend on the user to add a SaveModelCallback it's just that fastai makes it simpler to add callbacks before the training begins and manipulate them inside.

Reply:
I meant the SaveModelCallback if we are using it later then should lets leave it